src/Core/Ordering/Model/AddItemStrategy/MergeSameSkuItemsStrategy.php

```
@@ -0,0 +1,25 @@
+namespace Combee\Core\Ordering\Model\AddItemStrategy;
+
+use Combee\Core\Ordering\Contract\Model\AddItemStrategy\AddItemStrategyContract;
+use Combee\Core\Ordering\Contract\Model\OrderItemContract;
+use Doctrine\Common\Collections\Collection;
+
+final readonly class MergeSameSkuItemsStrategy implements AddItemStrategyContract
+{
+    public function addItem(Collection $items, OrderItemContract $newItem): void
+    {
+        $existingItem = $items->findFirst(
+            fn (int $key, OrderItemContract $item): bool => $item->productSku === $newItem->productSku
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)